internal/incident/incident.go

```diff
@@ -217,18 +163,27 @@ func (i *Incident) processSeverityChangedEvent(ctx context.Context, tx *sqlx.Tx,
 		}
 	}
 
+	i.Severity = newSeverity
+	if err := i.Sync(ctx, tx); err != nil {
+		i.logger.Errorw("Failed to update incident severity", zap.Error(err))
+
+		return causedByHistoryId, errors.New("failed to update incident severity")
+	}
+
 	return causedByHistoryId, nil
 }
 
 func (i *Incident) processIncidentOpenedEvent(ctx context.Context, tx *sqlx.Tx, ev event.Event) error {
 	i.StartedAt = ev.Time
+	i.Severity = ev.Severity
 	if err := i.Sync(ctx, tx); err != nil {
 		i.logger.Errorw("Can't insert incident to the database", zap.Error(err))
 
 		return errors.New("can't insert incident to the database")
 	}
 
-	i.logger.Info("Opened incident")
+	i.logger.Infow(fmt.Sprintf("Source %d opened incident at severity %q", ev.SourceId, i.Severity.String()), zap.String("message", ev.Message))
+
 	historyRow := &HistoryRow{
 		Type:    Opened,
 		Time:    types.UnixMilli(ev.Time),
```

Comment:
If the incident is now opened with the non-ok severity directly (which arguably makes more sense), the information that was previously contained in other history events should now be available in that history event (that is at least the new/initial severity and the message).